components/Button/button.css

```diff
@@ -0,0 +1,199 @@
+:root {
+
+    --primary-color: #38bdf8;
+    --primary-font: white;
+
+    --secondary-color: #f3f4f6;
+    --secondary-font: #38bdf8;
+
+    --disabled-font: darkgray;
+    --disabled-color: #d1d5db;
+
+    --outlined-bg-color: white;
+    --solid-border-color: white;
+
+
+    --ot-primary-font-color: #38bdf8;
+    --ot-primary-border-color: #38bdf8;
+
+    --ot-secondary-font-color: #9ca3af;
+    --ot-secondary-border-color: #9ca3af;
+
+    --primary-text:#38bdf8;
+    --secondary-text:#9ca3af;
+
+    --primary-hover-color:#7dd3fc;
+    --seconadry-hover-color:#f3f4f6;
+
+}
+
+
+
+.button-container.solid  {
+    display: grid;
+    grid-gap: 1rem;
+    padding: 1.5rem 1.5rem;
+    box-shadow: 1px 2px 5px 3px rgba(0, 0, 0, 0.1);
+    grid-template-columns: repeat(3, 5rem);
+}
+
+.button-container.outlined{
+    display: grid;
+    grid-gap: 1rem;
+    padding: 1.5rem 1.5rem;
+    box-shadow: 1px 2px 5px 3px rgba(0, 0, 0, 0.1);
+    grid-template-columns: repeat(2, 6rem);
+}
+
+
+.button-solid, .button-outlined {
+    border: 1px solid var(--solid-border-color);
+    border-radius: 0.2rem;
+    font-size: 0.8em;
+    margin: 0.2rem 0.2rem;
```

Comment:
You are using font-size in em??, i think you should use rem

Reply:
Actually, Tanay suggested to use em for font size in level 0.